packages/wp-comments/src/__tests__/__snapshots__/commentsHandler.test.ts.snap

```diff
@@ -75,7 +75,7 @@ Object {
 
 exports[`commentsHandler should handle replies correctly 3`] = `
 Object {
-  "areComments": true,
+  "isComments": true,
```

Comment:
I think `isComments` is correct if you consider "Comments" to be a proper noun like "Rolling Stones" or "eighties" 🙂